### ISSUE TITLE
games_services 3.0.3 may freeze on signOut (Android)

### ISSUE THREAD

**Issue by andynewman10** [Abedalkareem/games_services#130]
I am creating this issue which is only relevant for older versions of games_services, just in case it is appopriate to be tracked. Feel free to disregard/close or treat it as 'won't fix'.

Experiencing the issue on Android 7.1.1 and Android 13 Go.

Working scenario:

1) launch app
2) signIn() -> PGSv1 sign-in dialog appears. Click Authorize.
3) signOut() -> it works. Next time signIn() is ever called, the PGSv1 sign-in dialog will appear again.

Non-working scenario:

1) launch app
2) signIn() -> PGSv1 sign-in dialog appears. Click Authorize.
3) close the application
4) launch the application
5) signIn() -> sign-in takes place, without the PGSv1 dialog as you already authorized.
6) signOut() -> freezes. If you ignore the freeze using a .timeout() on the future, then open the Play Games app, you will see that signOut() did not work.

My setup: VS code, Windows, Flutter 3.16.0, everything run in USB debug sessions.

**Comment by Abedalkareem** [comment]
@theLee3 
I think this is fixed with the latest changes correct?

**Comment by theLee3** [comment]
Correct. This is not an issue with the recent versions of games_services. He is noticing it specifically on older versions using PGSv1. We can close.

**Comment by andynewman10** [comment]
Correct, closing the issue.
